### ISSUE TITLE
conjuncts.plx regression test fails

### ISSUE THREAD

**Issue by ajjimenezm** [plexil-rwl-support/PLEXIL5#10]
Looks like a parsing error

**Comment by Koliopkl** [comment]
For the arguments of lookups, we are using lists of values (val(x) val(y)). However, every time we are referring to PLEXIL values with type, such as StringValue or IntegerValue, we represent them as Expressions (const(val(x))). Should we consider changing it? Or should we modify the parser for specific cases in the arguments of lookups to remove the const() wrapper?

**Comment by Koliopkl** [comment]
@mfeliu 
I have made it so that lookups are parsed according to what is specified in the semantics, which as I said yesterday, is not consistent with how we are representing PLEXIL values. However, I have doubts about how stateLookups in scripts should be parsed, and the documentation did not help me.

As of right now, something like "state A ("is" : string) = 1 : int;" in a script is parsed as "stateLookup('A,(nilarg),val(1))". However, isn't the argument "is" being omitted? I was thinking it should be parsed as "stateLookup('A,(val("is")),val(1))", but I am not sure right now.

I have tried both representations for conjuncts. With the current representation there are 4 differences, and with my proposal there are 3. Of course, something in the semantics needs changing, but I didn't want to dabble with the semantics before knowing what you thought about changing the arguments and tolerance of lookups.
